x-pack/plugins/security_solution/public/explore/components/risk_score/enable_risk_score/index.tsx

```diff
@@ -49,7 +49,10 @@ const EnableRiskScoreComponent = ({
 
   return (
     <EuiPanel hasBorder>
-      <HeaderSection title={<RiskScoreHeaderTitle riskScoreEntity={entityType} />} titleSize="s" />
+      <HeaderSection
+        title={<RiskScoreHeaderTitle riskScoreEntity={entityType} showTechnicalPreviewBadge />}
```

Comment:
Label is present when risk score is not yet enabled

![image](https://user-images.githubusercontent.com/18648970/235977265-6a1e282d-e846-4d59-90cd-8d41238c8e43.png)